README.md

```diff
@@ -66,11 +40,48 @@ Build a small application exposing the following endpoints through a REST API:
 
 We'll evaluate:
  
- * the complexity & scalability of your algorithm
- * the readability of your code (including readability of your tests)
- * the correctness of the outputs of the API
- * you're ability to share your design choices and clearly weight pros & cons of alternative solutions
- * the overall quality of your written technical communication 
+ * The complexity & scalability of your algorithm
+ * The readability of your code (including readability of your tests)
+ * The correctness of the outputs of the API
+ * You're ability to share your design choices and clearly weight pros & cons of alternative solutions
+ * The overall quality of your written technical communication
+
+
+### Part 2: System design
+
+Describe briefly how you would build at scale and in the cloud (preferably with GCP) an API to power a simplified page of our Search Analytics:
+
+<img src="analytics-ui-example.png" alt="Simplified UI" width="400"/>
+
+For a given timerange, you must be able to return:
+* The top 1000 top popular searches
+* Their count
+* Their CTR (Click through rate)
+
+The new service(s) must fit in the following architecture:
+
+![Architecture](architecture.png)
+
+Some additional information:
+* The traffic expected is up to 100K searches and 10K click events per day per customer application
+* Each click is tied to a specific search in the log with a `queryID`
+* Click events are accepted by the API if they are reiceved up to 15 minutes after the search
+* Customer may request up to 3 month of timerange in the Analytics
+
+Describe the eventual databases that you would setup and their content, the eventual packaged services you would use, and the different processes you would create. For each logical brick, explain their goal and how do they communicate together.
+
+#### Evaluation Criteria
+
+* Please push it along with your code on GitHub repository or in the archive
+* Stay concise (maximum 1 page)
```

Comment:
Again here, I'd be happy with 2-3 pages especially if it includes several diagrams.